src/sentry/options/manager.py

```diff
@@ -77,7 +77,8 @@ def __init__(self, store):
         self.store = store
         self.registry = {}
 
-    def set(self, key, value, coerce=True):
+    # TODO: Find out why `from sentry.models.options import OptionsTypes` doesn't work at top level
+    def set(self, key, value, coerce=True, source="legacy"):
```

Comment:
```
# TODO: Find out why `from sentry.models.options import OptionsTypes` doesn't work at top level
```

What problem are you running into ?

Reply:
I believe it's an cyclical import or other import dependency that's causing this to not work. Some discussion [here](https://github.com/getsentry/sentry/pull/47510/files#r1171685519) that's related. I think I may pull the enum into the options module and then import from there into the models module. That's just an idea so I'd be interested in hearing others.